tsconfig.json

```diff
@@ -3,7 +3,7 @@
   "exclude": ["src/_shims/**/*-deno.ts"],
   "compilerOptions": {
     "target": "es2020",
-    "lib": ["es2020"],
```

Comment:
ah, sadly I don't think this is an acceptable change b/c we don't want to drag DOM types into our users' projects if they are in a Node setting. Do you remember why it was needed?

Reply:
`ReadableStream` isn't defined in the base typing - let me see if I can find another workaround.